docs/docs/migration_notes.md

```diff
@@ -259,6 +259,25 @@ For this reason we've decided to rename it:
 
 To reduce loading times, the package `@aztec/noir-contracts.js` no longer exposes all artifacts as its default export. Instead, it exposes a `ContractNames` variable with the list of all contract names available. To import a given artifact, use the corresponding export, such as `@aztec/noir-contracts.js/FPC`.
 
+### Blobs
+We now publish all DA in EVM blobs rather than calldata. This replaces all code that touched the `txsEffectsHash`.
```

Comment:
``` suggestion
We will soon publish all DA in L1 blobs rather than calldata. This replaces all code that touched the `txsEffectsHash`.
```

Reply:
we don't yet publish everything to blobs right? Thouygh obviously we have these relevant code changes!

Reply:
True, we will publish everything with #11632, I was just getting ahead of myself! I will make this more clear